src/pages/Header/Header.tsx

```diff
@@ -1,39 +1,179 @@
-import {Link} from "react-router-dom";
-
-function Header() {
-    return (
-        <header className="shadow-md p-6 bg-lime-300">
-
-            <div className="branding text-4xl font-extrabold text-center mb-6 text-indigo-500">
-                My App
-            </div>
-
-            <div className="text-center text-lg font-light mb-4 text-gray-600">
-                Become better !!!
-            </div>
-
-            <nav>
-                <ul className="flex justify-center space-x-6">
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500" to="/">Home</Link>
-                    </li>
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500"
-                              to="/new">Discover</Link>
-                    </li>
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500"
-                              to="/registration">Sign up</Link>
-                    </li>
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500"
-                              to="/authorization">Login</Link>
-                    </li>
-                </ul>
-            </nav>
-
-        </header>
-    );
-}
+import logo from "../../assets/images/logo.png";
+
+const Header = () => {
+  return (
+    <div className="bg-indigo-900 text-white">
+      <nav className="mx-auto flex flex-col lg:flex-row items-center justify-between p-6 lg:px-8">
+        <div className="lg:w-1/4 w-full flex items-center justify-center">
+          <img
+            src={logo}
+            alt="Healthcare logo"
+            className="mx-auto"
+            style={{ maxWidth: "130px", height: "auto", margin: "0" }}
```

Comment:
you can use classes for it

Reply:
@eugenchen read about tailwind, it will help you in future to develop quick
for ex. try to replace `style={{ maxWidth: "130px", height: "auto", margin: "0" }}` on `className="max-w-[130px] h-auto m-0"`